src/index.html

```diff
@@ -4,9 +4,320 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo"
+        >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#Apple" class="nav__link is-active">
+              apple
+            </a>
+          </li>
```

Comment:
Better rename(the same naming you could use for projects in future). Rename all `nav items`
Add changes to styles according to changes in html.
```suggestion
          <li class="nav__item">
            <a href="#Apple" class="nav__link is-active">
              apple
            </a>
          </li>
```